EzPC/codegen.ml

```diff
@@ -76,7 +76,7 @@ let o_cbfunction :secret_label -> comp -> comp list -> comp = o_cbfunction_maybe
 let o_sunop (l:secret_label) (op:unop) (c:comp) :comp =
   let c_op =
     match op with
-    | U_minus -> failwith "Codegen: unary minus should have been desugared"
+    | U_minus -> failwith "Codegen: unary minus is not being produced by lexer or parser right now."

```

Comment:
Either we should support unary minus syntax, or add a comment in `ast.ml` OR `parser.mly` that we don't.